card/src/main/java/com/adyen/checkout/card/data/ExpiryDate.java

```diff
@@ -10,14 +10,17 @@
 
 public class ExpiryDate {
     public static final int EMPTY_VALUE = 0;
-    public static final ExpiryDate EMPTY_DATE = new ExpiryDate(EMPTY_VALUE, EMPTY_VALUE);
+    public static final ExpiryDate EMPTY_DATE = new ExpiryDate(EMPTY_VALUE, EMPTY_VALUE, false);
+    public static final ExpiryDate EMPTY_DATE_WITH_INPUT = new ExpiryDate(EMPTY_VALUE, EMPTY_VALUE, true);
```

Comment:
Maybe we could habve `EMPTY_DATE` and `INVALID_DATE` instead of adding the boolean flag.

Reply:
done